### PR TITLE
upgrades to php7.4

### PR DESCRIPTION
**Pull request by raftx24**

### CODE REVIEW

**Comment by checkerschaf** [comment]
Why to force 7.4 that early?

**Comment by raftx24** [comment]
because we use arrow functions.

**Comment by checkerschaf** [comment]
I saw your commit but it's not used yet, is it?

**Comment by aocneanu** [comment]
We will probably publish enso 3.8.0 ~ 28 Dec. it won’t be merged until then.

File: src/app/Traits/HistoryTracker.php [hunk 1]
```diff
@@ -10,13 +10,9 @@ trait HistoryTracker
...
     {
-        self::created(function ($model) {
-            $model->saveHistory();
-        });
+        self::created(fn($model) => $model->saveHistory());
 
-        self::updated(function ($model) {
-            $model->saveHistory();
-        });
+        self::updated(fn($model) => $model->saveHistory());
```
Comment (raftx24) [inline src/app/Traits/HistoryTracker.php]: @checkerschaf we use it here.

Comment (checkerschaf) [reply src/app/Traits/HistoryTracker.php]: @raftx24 I know that you use it NOW but not YET. In my opinion it's not a big improvement to use it at the moment but I don't mind.

**Pull request merged**